tests/Fixtures/e2e/Test_Frameworks/spec/SourceClassSpec.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace spec\Namespace_;
+
+use PhpSpec\ObjectBehavior;
+
+class SourceClassSpec extends ObjectBehavior
+{
+    public function it_returns_hello()
```

Comment:
not a big deal to have mistakes in fixture classes, but still - `hello`? probably copy paste issue 

two lines below - `1, 2` - space after comma

Reply:
Ah my bad, that is indeed a coppy paste issue.